### PR TITLE
Added e2e playground and some remarks.

### PR DESCRIPTION
**Pull request by chaosmail**
Finally I added the flexible playground as an e2e template. We can use it for loads of tests now.

### CODE REVIEW

File: src/LineChart.ts [hunk 1]
```diff
@@ -101,13 +101,18 @@ module n3Charts {
       });
 
       // Trigger the resize event
-      angular.element(this.$window).on('resize', (event: UIEvent) => {
+      // Added <any> cast for TS linter error, needs further investigation
+      // [chaosmail]
+      angular.element(this.$window).on(<any>'resize', (event: UIEvent) => {
```
Comment (lorem--ipsum) [inline src/LineChart.ts]: Hmmm TypeScript...

File: src/styles/_legend.scss [hunk 2]
```diff
@@ -19,6 +19,9 @@
       display: inline-block;
     }
 
+    // We should consider using a prefix here - like n3-label -
+    // due to naming collision with bootstrap
+    // [chaosmail]
```
Comment (lorem--ipsum) [inline src/styles/_legend.scss]: Yup, agreed

File: src/styles/_legend.scss [hunk 3]
```diff
@@ -33,6 +36,9 @@
       background-position: 50% 25%;
     }
 
+    // We should consider using a prefix here - like n3-hidden -
+    // due to naming collision with bootstrap
+    // [chaosmail]
```
Comment (lorem--ipsum) [inline src/styles/_legend.scss]: Yup, agreed

**Comment by lorem--ipsum** [comment]
:poultry_leg:

**Comment by chaosmail** [comment]
man, that was quick :shipit: 
the playground is here mainly to define the options and settings we want to implement and then try them - manually and per e2e tests.
for CSS, we should consider prefixing all class names - not just the ones the collide

**Comment by lorem--ipsum** [comment]
Yeah, I think at one point we'll refresh the CSS file(s) so it's ready for "prod"

**Pull request merged**